resources/lang.js

```diff
@@ -176,6 +176,8 @@ class CactbotLanguage {
       EurekaPyros: /(Eureka Pyros|Unknown Zone \(31B\))/,
       EurekaHydatos: /(Eureka Hydatos|Unknown Zone \(33B\))/,
     });
+
+    callOverlayHandler({ call: 'cactbotRequestPlayerUpdate' });
```

Comment:
Also, I'm not quite sure I understand why there's a cactbotRequestPlayerUpdate and a cactbotRequestState separately.  I would have expected to just have a single "the JS is ready send everything again" message.

Reply:
I wasn't entirely sure in which order the events were sent and the code was executed. This is just me being paranoid and making sure that the `onPlayerChangedEvent` happens at the correct time to avoid the `gLang.countdownEngageRegex` error.
I think this constructor is only called from `getUserConfigLocation` so it should be fine if I remove this call since the whole notification state is reset there.